src/helpers.js

```diff
@@ -0,0 +1,79 @@
+//!Defines two helper functions.
+
+/*
+ * c
+ * https://github.com/rumpl/c
+ *
+ * Copyright (c) 2012 Djordje Lukic
+ * Licensed under the MIT license.
+ */
+
+"use strict";
+var underscore = require("underscore");
+var helpers = module.exports;
+var colors = require("colors"); //Despite looking unused, is not unused.
```

Comment:
This could be `require("colors");`, no need to define a variable to take it.

Or we could use the safe one that doesn't change the `String` prototype: 
```javascript
var colors = require('colors/safe');
consle.log(colors.brightGreen('something'));
```